src/viewer.js

```diff
@@ -334,8 +334,19 @@ function _updateFeatureMeasurements (map, feature, pyramid) {
   }
 
   const measurements = feature.get(Enums.InternalProperties.Measurements) || []
-  const area = getFeatureScoord3dArea(feature, pyramid)
-  const length = getFeatureScoord3dLength(feature, pyramid)
+
+  let area, length;
+  try {
+    area = getFeatureScoord3dArea(feature, pyramid)
+  } catch {
+    area = 0;
+  }
+  
+  try {
+    length = getFeatureScoord3dLength(feature, pyramid)
+  } catch {
+    length = 0;
+  }
```

Comment:
I am not sure this is the way to go due to the following two concerns:

1. These concepts don't make sense for all graphic types. For example, a POINT or MULTIPOINT doesn't have an area or a length, a POLYLINE doesn't have an area, and a POLYGON or ELLIPSOID doesn't have a length. Setting the value to `0` may be misleading and could lead to bugs down the line. In this case, I would prefer assigning `undefined`, `null`, or `NaN` and handle the value explicitly downstream (`0`, `null`, `undefined`, and `NaN` are all [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)).

2. `try...catch` blocks are brittle in JavaScript in my experience. I tend prefer the C approach and return `undefined`/`null` if the value cannot be computed for the given type and check the return value rather than throwing and catching an exception. What do you think?

Reply:
So the idea to duplicate the ifs that live inside each get function? or we should remove the exception + move the if outside the function?

Reply:
There are currently two exceptions thrown. I would keep the first (missing coordinate values), but get rid of the second exception and return `null` or `undefined` instead. Here, we could just add `if (area !== undefined)` as a check before adding a measurement.

Reply:
I'll update accordingly.

Reply:
I would suggest using an explicit boolean expression to be on the safe side.